app/src/composables/useSiwe.ts

```diff
@@ -10,17 +9,17 @@ import { getFetchErrorMessage, log, parseError } from '@/utils'
 import { useCustomFetch } from './useCustomFetch'
 import { useToastStore } from '@/stores/useToastStore'
 import { MetaMaskUtil } from '@/utils/web3Util'
+import { useStorage } from '@vueuse/core'
 
 const ethersJsAdapter = EthersJsAdapter.getInstance() //new EthersJsAdapter()
-const siweAuthApi = new SiweAuthAPI()
 
 function createSiweMessageCreator(address: string, statement: string, nonce: string | undefined) {
   return new SLSiweMessageCreator({
     address,
     statement,
     nonce,
     version: '1',
-    chainId: '1'
+    chainId: 1
```

Comment:
The `version` and `chainId` are currently hard coded they need to be queried from MetaMask

Reply:
OK thanks.